src/domain/checkForAndPlaceOrder.ts

```diff
@@ -424,55 +424,41 @@ export const _printUnfilledOrders = (orders: Map<BytesLike, OrderStatus>) => {
 async function _placeOrder(params: {
   orderUid: string;
   order: any;
```

Comment:
I think this order should have a type (and be build ourside of this method). 
The SDK should expose the different Order types, probably this one is the `OrderCreation`?

Reply:
I looked at this a bit, and I think we need to bump the `balanceToString`, `kindToString` methods etc from the watch tower to the `cow-sdk`. `kindToString` is already within the `composable` section, but it's not exported. I think we include this as part of #18

Reply:
yes, agree